### PR TITLE
Update LICENSE-NEURALMAGIC

### PR DESCRIPTION
**Pull request by rgreenberg1**
Updated with name change, updated date

### CODE REVIEW

File: LICENSE-NEURALMAGIC [hunk 1]
```diff
@@ -1,11 +1,11 @@
-                  NEURAL MAGIC ENGINE COMMUNITY LICENSE AGREEMENT
+                  NEURAL MAGIC DEEPSPARSE COMMUNITY LICENSE AGREEMENT
```
Comment (jeanniefinks) [inline LICENSE-NEURALMAGIC:1]:
```suggestion
Updated: November 16, 2022

NEURAL MAGIC DEEPSPARSE COMMUNITY LICENSE AGREEMENT
```

File: LICENSE-NEURALMAGIC [hunk 2]
```diff
@@ -67,6 +67,6 @@ license@neuralmagic.com.
 Your rights under this Agreement will terminate automatically without notice if you fail to 
 comply with any term(s) of this Agreement.
 
-Updated August 1, 2022
+Updated November 16, 2022
```
Comment (jeanniefinks) [inline LICENSE-NEURALMAGIC:70]:
```suggestion
```

File: LICENSE-NEURALMAGIC [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+Please read this Neural Magic DeepSparse Community License Agreement (“Agreement”) 
 carefully before using Software for the first time. By using the Software, you agree 
 to these terms. If you do not agree to the terms of this Agreement, the Software may 
 not be installed on your computer systems. 
 
-The DeepSparse Engine (“Software”) consists of a current version of: (a) machine 
+DeepSparse (“Software”) consists of a current version of: (a) machine 
 executable (object) code, and (b) any existing user guide and release notes. The 
 Software is copyrighted by Neuralmagic, Inc. (“Neural Magic”), a Delaware corporation 
 with offices at 55 Davis Square STE 3, Somerville, MA 02144, and is licensed and not
```
Comment (jeanniefinks) [inline LICENSE-NEURALMAGIC:13]:
```suggestion
with offices at 55 Davis Square STE 3, Somerville, MA 02144 United States, and is licensed and not 
```

File: LICENSE-NEURALMAGIC [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
+NEURAL MAGIC DEEPSPARSE COMMUNITY LICENSE AGREEMENT
+
+Please read this Neural Magic DeepSparse Community License Agreement (“Agreement”) 
 carefully before using Software for the first time. By using the Software, you agree 
 to these terms. If you do not agree to the terms of this Agreement, the Software may 
 not be installed on your computer systems. 
 
-The DeepSparse Engine (“Software”) consists of a current version of: (a) machine 
+DeepSparse (“Software”) consists of a current version of: (a) machine 
 executable (object) code, and (b) any existing user guide and release notes. The
```
Comment (jeanniefinks) [inline LICENSE-NEURALMAGIC:11]:
```suggestion
executable (object) code, (b) DeepSparse-branded components, and (c) any existing user guide and release notes. The 
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
Assuming we can generically refer to Server, Pipeline, and Benchmark as "DeepSparse-branded" components if they are considered part of the Software. It's left intentionally generic to serve as umbrella to future components as well as potential name changes.

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
@rgreenberg1 also update 
"or any purpose." to
"for any purpose."

**Review by jeanniefinks — DISMISSED** [review]

**Review by bnellnm — DISMISSED** [review]

**Review by jeanniefinks — APPROVED** [review]

**Review by bnellnm — APPROVED** [review]

**Pull request merged**